From: Paul Boddie <paul@boddie.org.uk>
Date: Tue, 15 Aug 2023 23:50:33 +0200
Subject: Remove setuptools_scm retrieval

Since the python3-setuptools-scm package is a build dependency, setup.py
should not attempt to retrieve the package from the Python Package Index
itself.

moin (2.0+git20230330.c55318e3-1) unstable; urgency=medium

  * Initial release.
---
 setup.py | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/setup.py b/setup.py
index 583b32c..78edb5d 100644
--- a/setup.py
+++ b/setup.py
@@ -67,9 +67,6 @@ setup_args = dict(
     use_scm_version={
         'write_to': os.path.join(basedir, 'src', 'moin', '_version.py'),
     },
-    setup_requires=[
-        'setuptools_scm',  # magically cares for version and packaged files
-    ],
     install_requires=[
         'Babel>=2.10.0',  # internationalization support
         'blinker>=1.1',  # event signalling (e.g. for change notification trigger)
